runtime/lua/vim/_meta.lua

```diff
@@ -603,6 +610,17 @@ local create_option_metatable = function(set_type)
       return convert_value_to_lua(self._name, self._info, self._value)
     end,
 
+    default = function(self)
+      return convert_value_to_lua(self._name, self._info, self._info.default)
+    end,
+
+    reset = function(self)
+      local value = convert_value_to_vim(self._name, self._info, self._info.default)
```

Comment:
can this be `local value = self:default()` ?

Reply:
No, `:default()` returns the value as a Lua object (`convert_value_to_lua`) while this needs to use the Vim script value (`convert_value_to_vim`) to pass to `nvim_set_option_value`.